README.md

```diff
@@ -49,6 +49,7 @@ The csv format looks as follows: `Type;Name;Site;TargetSite;Permanent;Regex;Prot
 |Term|Defines the term that should be matched|
 |Target|Defines the target of the redirect. This can be either an external URL (eg. `https://google.com`), or the path to an item. Please note that the path shall not include any language code or site name (if you are using virtual paths). Example: `/products/super-product`|
 |Description|An optional description of the redirect. This will be fed into the `Description` field on the created redirect item|
+|PreserveQueryString|Adds query string regex pattern to the term field. Set to `x` to handle and pass along query string from the requested URL. It also enables support for regular expressions|
```

Comment:
**question:** What does the documentation highlight with this last sentence:
> It also enables support for regular expressions

**thought:** If it is an implementation detail ("to enable preservation of the query string, we use the existing regex functionality"), then I would remove it here. The implementation can always change and it's not adding much value to the end user that tries to understand the flag.

**suggestion:** I would most likely simplify the documentation to the essential part:
> Set to `x` to handle and pass along query string from the requested URL.

Reply:
Agree